support/cas-server-support-cassandra-ticket-registry/src/test/java/org/apereo/cas/utils/TicketCreator.java

```diff
@@ -20,7 +20,7 @@
  *
  * @since 5.1.0
  */
-public class TicketCreator {
+public final class TicketCreator {
```

Comment:
`public final class TicketCreatorUtils` is the common idiom. We also have a few others such as `Beans` etc. If you wish to explicitly make a utility class, you likely should postfix it with Utils.